cosmos/lib/cosmos/api/interface_api.rb

```diff
@@ -29,10 +29,8 @@ module Api
       'get_interface_names',
       'connect_interface',
       'disconnect_interface',
-      # DEPRECATED:
-      'interface_state',
-      'get_interface_targets',
-      'get_interface_info',
+      'start_raw_logging_interface',
+      'stop_raw_logging_interface',
```

Comment:
I think it makes sense to add these methods here rather than in a separate logging_api.